assignment-05.Rmd

```diff
@@ -1,145 +1,269 @@
-
-    ``` {.r}
-    J <- matrix(c(df_dx, df_dy, 
-                  dg_dx, dg_dy), 
-                ncol = 2, nrow = 2, 
-                byrow = TRUE)
-
-    e_vals <- eigen(J)$values
-    ```
-
-    For each point, give the eigenvalues and explain whether the fixed
-    point is stable or unstable. Will it oscillate? Explain why or why
-    not (1 marks).
-
-(e) Simplify the fixed points of the system on paper in the case when
-    $\alpha_1 = \alpha_2 = \alpha$, $r_1 = r_2 = r$, and $K_1 \neq K_2$.
-    Create a plot in R showing $x^*$ vs. $\alpha$, $y^*$ vs $\alpha$,
-    and $x ^ * + y ^ *$ vs. $\alpha$ for \$ -1 &lt; \alpha &lt; 1.5\$
-    for the *non-trivial* fixed point where both $x^*$ and $y^*$ are
-    not zero. Note that $\alpha = -1$ and $\alpha = 1$ will give
-    infinity, so skip those two points when plotting. Label $K_1$,
-    $K_2$, and $K_1 + K_2$ on the y-axis. (2.5 marks total)
-
-    -   Thinking of $\alpha$ as the parameter that controls "overlap"
-        between the two competing species, what is the biological
-        meaning of $\alpha = 0$? (Hint: compare this fixed point to the
-        non-trivial fixed point from the logistic model: $N^* = K$.)
-
-    -   What is the biological meaning of $0 < \alpha < 1$? What is the
-        biological meaning of $\alpha = 1$, and of $\alpha > 1$? (Hint:
-        what is the sum of the non-trivial fixed points $x^*$ and $y^*$
-        in each of these cases in terms of $K_1$ and $K_2$?)
-
-    -   What would happen to the system if $\alpha$ was negative? (Hint:
-        is "competition" still an appropriate name for the model if
-        $\alpha < 0$?)
+including the original questions, your code, and the output. Submit
+you assignment as a knitted `.pdf` (prefered) or `.html` file.*
+
+**1.  Linear mixed-effects models (4 marks).**
+
+Santangelo _et al._ (In press) were interested in understanding how plant defenses, herbivores,
+and pollinators influence the expression of plant floral traits (e.g. flower size). Their experiment
+had 3 treatments, each with 2 levels: Plant defense (2 levels: defended vs. undefended), herbivory
+(2 levels: reduced vs. ambient) and pollination (2 levels: open vs. supplemental). These treatments
+were fully crossed for a total of 8 treatment combinations. In each treatment combination, they grew 
+4 individuals from each of 25 plant genotypes for a total of 800 plants (8 treatment combinations x 
+25 genotypes x 4 individuals per genotype). Plants were grown in a common garden at the Koffler 
+Scientific Reserve (UofTs field research station) and 6 floral traits were measured on all plants 
+throughout the summer. We will analyze how the treatments influenced one of these traits in this 
+exercise. Run the code chunk below to download the data, which includes only a subset of the columns 
+from the full dataset:
+    
+```{r}
+plant_data <- readr::read_csv("data/Santangelo_JEB_2018.csv", header = TRUE)
```

Comment:
Seems `header = TRUE` is breaking Travis - I think the `readr` equivalent is `col_names`, but it's also set to `TRUE` by default, so no second argument is needed